bin/runtime/src/lib.rs

```diff
@@ -104,19 +107,21 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 8,
+    spec_version: 9,
```

Comment:
most probably you will be merging after #307 so remember to bump this to `10`

Reply:
bumped.